book/graphics.md

```diff
@@ -28,13 +28,6 @@ desktop environment controls the screen. Therefore:
 -   The desktop environment tells the program about clicks and key
     presses, and the program responds and redraws its window.
 
-[^sixty-hertz]: Most screens today have a refresh rate of 60Hz, and
-that is generally considered fast enough to look smooth. However, new
-hardware is increasingly appearing with higher refresh rates, such as
-120Hz. It's not yet clear if browsers can be made that fast. Some
-rendering engines, games in particular, refresh at lower rates on
-purpose if they know the rendering speed can't keep up.
-
```

Comment:
Footnote moved lower down. The only actual change is adding a non-breaking space between the figure and unit in "60Hz".